src/resources/uart_esp32.cc

```diff
@@ -1016,6 +1043,11 @@ PRIMITIVE(get_control_flags) {
   FAIL(UNIMPLEMENTED);
 }
 
+PRIMITIVE(errors) {
+  ARGS(UartResource, uart)
+  return Smi::from(uart->errors());
```

Comment:
How do you want this to behave if the error count is outside the smi-range?

Reply:
Changed it to `Primitive::Integer...`
done.